csharp/ql/test/query-tests/Bad Practices/Implementation Hiding/StaticArray/StaticArray.cs

```diff
@@ -16,6 +16,8 @@ class Program
 
     public static readonly int[] NonEmptyArray3; // BAD
 
+    public static readonly int[,,,] NonEmptyArray4 = new int[2, 3, 0, 2]; // BAD
```

Comment:
Isn't this also empty? I mean along the 3rd dimension it's length is 0, so it can contain no data, can it?

Reply:
I did think about this too, but it could be replaced with another empty array, which would be a different object.

Reply:
I don't know how you could do that. 
This doesn't work:
```
var x = new int[2,0]{ {}, {} };
x.SetValue(new int[0], 0, 0); // System.IndexOutOfRangeException: Index was outside the bounds of the array.
```
And I don't think you could do partial indexing, like `x[0] = ...`.

Reply:
Ah, you are absolutely right, sorry. I got confused and thought of multi-dimensional arrays as arrays of arrays. So closing this PR again.